jobs/build/sync-for-ci/Jenkinsfile

```diff
@@ -127,15 +131,28 @@ node {
 
                 stage("push to s3") {
                     commonlib.syncRepoToS3Mirror("${LOCAL_SYNC_DIR}/", "${S3_SYNC_DIR}/")
+                    if ( commonlib.ocp4Versions.get(0) == REPOSYNC_DIR ) {
```

Comment:
This assumes versions in `commonlib.ocp4Versions` are in descending order. It's ok for now but I am afraid this will bite us someday.
Better to loop over this array to determine which one is the latest.